pkg/controller/common/http.go

```diff
@@ -0,0 +1,11 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package common
+
+const (
+	InternalProductRequestHeaderString = "x-elastic-product-origin: cloud"
```

Comment:
Nit: why are we not just concatenating key and value with a colon at the place where we need both instead of duplicating the content in the `*String` version of the constant?